### ISSUE TITLE
Button in Hamburger Menu not working correctly + Lack of focus trap

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#46]
[Aan de slag met API's](https://don.apps.digilab.network/kennisbank/apis/aan-de-slag/)

Het Hamburger menu opent en toont de inhoud van de Sidebar (Aan de slag met API's) in plaats van de Hoofdnavigatie (Home, API's, etc.). Je kunt alleen toegang krijgen tot de hoofdnavigatie vanuit het hamburger menu via de "<- Terug naar het hoofdmenu" knop. De focus beweegt visueel naar het hoofdmenu maar tabben brengt je terug naar de inhoud van de sidebar (en niet naar de items van de hoofdnavigatie). Alleen door achteruit te tabben kun je het Hoofdmenu betreden.

Opmerking: Voortdurend achteruit tabben door de inhoud van de Hoofdnavigatie brengt de focus uiteindelijk buiten het hamburger menu (door een gebrek aan focus trap). Als men dan vooruit tabt totdat men weer landt op de "<- Terug naar het hoofdmenu" knop en probeert erop te klikken, zal dit niet meer werken.

`<button type="button" class="clean-btn navbar-sidebar__back">← Terug naar het hoofdmenu</button>`

WCAG 2.4.3 Focus order

<img width="1004" alt="Hamburger menu with unexpected tabbing order" src="https://github.com/user-attachments/assets/34754730-f25c-4d23-8b85-20d29488d73b" />

_Reference: A041_
